diff --git a/node_modules/next/dist/build/adapter/build-complete.js b/node_modules/next/dist/build/adapter/build-complete.js
index 13d84ab..39f4635 100644
--- a/node_modules/next/dist/build/adapter/build-complete.js
+++ b/node_modules/next/dist/build/adapter/build-complete.js
@@ -618,7 +618,7 @@ async function handleBuildComplete({ dir, config, appType, buildId, configOutDir
             };
             let prerenderGroupId = 1;
             const getAppRouteMeta = async (route, isAppPage)=>{
-                const basename = route.endsWith('/') ? `${route}index` : route;
+                const basename = (0, _normalizepagepath.normalizePagePath)(route);
                 const meta = isAppPage ? JSON.parse(await _promises.default.readFile(_path.default.join(appDistDir, `${basename}.meta`), 'utf8').catch(()=>'{}')) : {};
                 if (meta.headers) {
                     // normalize these for consistency
